packages/terminal/src/browser/terminal-widget-impl.ts

```diff
@@ -306,6 +316,28 @@ export class TerminalWidgetImpl extends TerminalWidget implements StatefulWidget
         this.term.options.fastScrollSensitivity = this.preferences.get('terminal.integrated.fastScrollSensitivity');
     }
 
+    protected isTerminalAnsiColor(color: string): boolean {
```

Comment:
If I understand correctly, the icon is not in the terminal, but in the widget title. Why would we go through the terminal theme service here?

Reply:
@tsmaeder 
I'm not quite sure what you mean. But I think in this function the author checks if the configured color is one of the predefined colors from `packages/terminal/src/browser/terminal-theme-service.ts` that are preconfigured in `terminalAnsiColorMap`.
What would you suggest to change here?

Reply:
Why do we care whether this is a in the terminal-theme-service? The color is the color of a ThemeIcon in theia.d.ts. That should go through the regular color registry like with the QuickInputItem stuff.

Reply:
The code is now using the regular color registry.